### ISSUE TITLE
Can't run a simple Hello Word in Alpine Linux (netcoreapp2.1)

### ISSUE THREAD

**Issue by AreYouQQ** [dotnet/coreclr#18231]
![virtualbox_2018-06-01_16-07-56](https://user-images.githubusercontent.com/37007704/40829754-038364d6-65b7-11e8-9841-b5b3ac262525.jpg)

SDK Version: 2.1.300
In Windows OS,I build it and It run OK,
First, Alpine Linux use the newest Alpine version .net core runtime 
But I put all files in Release Directory to Alpine Linux,When I run it, I get a error

**Comment by janvorli** [comment]
Based on the log above, you need to install the libintl package (apk add libintl)

**Comment by AreYouQQ** [comment]
![virtualbox_2018-06-01_17-30-33](https://user-images.githubusercontent.com/37007704/40834081-686e7650-65c2-11e8-91fd-5ac60adee482.jpg)

after I run "Apk add libintl ",
It also has error

**Comment by AreYouQQ** [comment]
when I "apk add icu" ,it run OK
Thanks

**Comment by janvorli** [comment]
You need to install all the prerequisite packages: icu, openssl, krb5, curl, zlib. Optionally lttng-ust too.

**Comment by janvorli** [comment]
The openssl, krb5 and curl are needed if you use HTTPS / crypto stuff.

**Comment by AreYouQQ** [comment]
Thanks
I think Microsoft dotnet (http://dot.net) website alphine install document need to add your help.

**Comment by janvorli** [comment]
@AreYouQQ right, I have not noticed that that site is missing details on Alpine installation. I'll make sure we fix that.

**Comment by janvorli** [comment]
@richlander the issue here is that on the http://dot.net site, we don't list dependencies that need to be installed on Alpine Linux to be able to use .NET Core. We don't even list them on https://docs.microsoft.com/cs-cz/dotnet/core/linux-prerequisites?tabs=netcore30.
So unless people use our docker images, they are clueless on what to install.
Could you please look into adding such doc or find someone who could do that? I am assigning this issue to you for now.
